### ISSUE TITLE
[Desktop] Download history is not cleared on exit

### ISSUE THREAD

**Issue by rEes9P** [brave/brave-browser#10013]
## Description 
_Clear browsing data > On exit > Download history_ doesn't work

## Steps to Reproduce
   1. Select option _Clear browsing data > On exit > Download history_ in Settings (do not select any of the other options)
   2. Download a file. Quit Brave. Reopen Brave.

## Actual result:
Download history is not cleared on exit

## Expected result:
Would expect download history to be cleared on exit

## Reproduces how often: 
Easily reproduced

## Brave version (brave://version info)
Brave: 1.9.76 Chromium: 81.0.4044.138 (Official Build) (64-bit)
Revision: 8c6c7ba89cc9453625af54f11fd83179e23450fa-refs/branch-heads/4044@{hashtag999}
OS: macOS Version 10.14.6 (Build 18G5033)

## Version/Channel Information:
- Can you reproduce this issue with the current release? Yes

## Miscellaneous Information:
Interestingly, if the option _Clear browsing data > On exit > Browsing history_ is selected in addition to _Download history_ then the Download history is cleared. However, the selection of Browsing history shouldn't have a bearing on whether Download history is cleared or not.

**Comment by rebron** [comment]
cc: @LaurenWags When you get a chance, can you see if you can reproduce? 
I wasn't able to using 1.11.x RC:

Brave | 1.11.94 Chromium: 84.0.4147.85 (Official Build) (64-bit)
-- | --
Revision | e20c13e958edc3707db3d26d513780a7f6331a4b-refs/branch-heads/4147@{#837}
OS | macOS Version 10.16 (Build 20A4300b)

![Screen Shot 2020-07-10 at 10 27 20 AM](https://user-images.githubusercontent.com/5951041/87181987-1c124b80-c298-11ea-872b-962d95e4924c.png)

@rEes9P We might have this already fixed as I'm not able to reproduce in our latest RC that's going out next week.

**Comment by LaurenWags** [comment]
@rebron duplicate of https://github.com/brave/brave-browser/issues/4004

**Comment by LaurenWags** [comment]
Does not reproduce using below build for me

```
Brave | 1.11.94 Chromium: 84.0.4147.85 (Official Build) (64-bit)
-- | --
Revision | e20c13e958edc3707db3d26d513780a7f6331a4b-refs/branch-heads/4147@{#837}
OS | macOS Version 10.14.6 (Build 18G3020)
```

Tried steps from description and steps from #4004

**Comment by rebron** [comment]
Closing as dupe of #4004 @mkarolin This looks fixed in 1.11.x? Can you confirm? Neither @LaurenWags or me could reproduce.
